src/main/kotlin/study/step3/domain/Car.kt

```diff
@@ -0,0 +1,15 @@
+package study.step3.domain
+
+class Car {
+    private var position = 0
+
+    fun move(shouldMove: Boolean) {
+        if (shouldMove) {
+            position++
+        }
+    }
+
+    fun getPosition(): Int {
+        return position
+    }
```

Comment:
```suggestion
    var position = 0
      private set

    fun move(shouldMove: Boolean) {
        if (shouldMove) {
            position++
        }
    }
```
var 로 선언된 변수를 외부에서 변경하는것을 막기 위해 일반적으론 위와 같이 `private set` 을 활용하곤 합니다 😄

Reply:
이걸로 바꿔보겠습니다!!